refactor/urls.js

```diff
@@ -1,16 +1,21 @@
-// Refactor this code
-
+//Refactor the code
 function getURL(dev = false, tasks = {}) {
-    let url = dev
-        ? `/tasks?status=ACTIVE&dev=true&size=20`
-        : '/tasks';
-
-    if (tasks.nextTasks) {
-        url += '?hasNext=true';
-    }
-
-    if (tasks.prevTasks) {
-        url = '/tasks?hasPrev=true';
-    }
-    return { url };
+  const { prevTasks = false, nextTasks = false } = tasks;
+  const baseUrl = "/tasks";
+  const params = {
+    status: "ACTIVE",
+    dev: "true",
+    size: "20",
+  };
+  const queryParams = new URLSearchParams(dev ? params : {});
+  if (nextTasks) {
+    queryParams.set("hasNext", "true");
+  }
+  if (prevTasks) {
+    queryParams.set("hasPrev", "true");
+  }
+  const url = queryParams.toString()
+    ? `${baseUrl}?${queryParams.toString()}`
+    : baseUrl;
```

Comment:
Possible to write this without conditionals?

Reply:
Yes, definetly this also works.
 const url = `${baseUrl}${queryParams.toString() && `?${queryParams.toString()}`}`;